Exercise_D_E/index.html

```diff
@@ -1,16 +1,62 @@
 <!DOCTYPE html>
 <html class="no-js">
-  <head>
-    <meta charset="utf-8" />
-    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-    <title></title>
-    <meta name="description" content="" />
-    <meta name="viewport" content="width=device-width, initial-scale=1" />
-    <link rel="stylesheet" href="./assets/style/index.css" />
-  </head>
+
+<head>
+  <meta charset="utf-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <title>Exercise D & E</title>
+  <meta name="description" content="" />
+  <meta name="viewport" content="width=device-width, initial-scale=1" />
+  <link rel="stylesheet" href="./assets/style/index.css" />
+</head>
+
+<body>
+  <!-- CODE HERE -->
+
   <body>
-    <!-- CODE HERE -->
+
+    <header> <img src="assets/images/logo.png" alt="logo"></header>
+
+
+    <div class="intro">
+      <h1> Beautiful Flowers </h1>
+
+      <button> Learn more </button>
+      <div class="background">
+
+
+      </div>
+    </div>
+
+
+    <div class="info">
+
+      <table class="table">
```

Comment:
Hello Chloe
Even though this layout does sort of work, i would suggest you can make use of Grid or Flex to add in items with Images and Text when laying them out on your page. You can look at the answers, if you would like an explanation for this we can look through it in a session later this week.